### ISSUE TITLE
iOS export error: “App installation failed: Could not inspect application package”

### ISSUE THREAD

**Issue by matheuslincolnss** [kloder-games/godot-admob#35]
Target iOS

Godot version: 3.0.2

Using admob module and exporting the godot3 example I had this error after include all frameworks:

![screen shot 2018-06-14 at 17 00 04](https://user-images.githubusercontent.com/12600520/41436635-9f5a85b0-6ff8-11e8-9697-9b899e2d77b6.png)

**Comment by Shin-NiL** [comment]
Did you tried [this](https://stackoverflow.com/a/30859736)?

**Comment by matheuslincolnss** [comment]
Hi man, I am brasileiro também, teria uma meio de trocarmos ideia, estou iniciando no godot.

**Comment by matheuslincolnss** [comment]
I already did it, but it stay failing

**Comment by Shin-NiL** [comment]
Pela falta de tempo, não sei se conseguiria ajudar muito individualmente. Se for sobre os módulos, o mais fácil seria no github mesmo, sobre assuntos gerais, no [fórum do Godot](https://godotdevelopers.org/) ou [Twitter](https://twitter.com/shin_nil).

**Comment by Shin-NiL** [comment]
Is there any special character in your project name?

**Comment by matheuslincolnss** [comment]
Nop, It is only a copy of godot-3 example and exported using the name GodotAd

![screen shot 2018-06-14 at 18 16 21](https://user-images.githubusercontent.com/12600520/41438884-343600c8-6fff-11e8-99d7-2f2f2342d4d9.png)

**Comment by Shin-NiL** [comment]
Are you able to run the app without the Admob module?

**Comment by matheuslincolnss** [comment]
yes

**Comment by matheuslincolnss** [comment]
Yes, I replaced the export template (.a) to another and then the project runs.
Export again and add the frameworks step by step and the error persists.

**Comment by matheuslincolnss** [comment]
Do you have a project on xcode working to share?

**Comment by Shin-NiL** [comment]
Nope, I'm sorry but I don't even have a Mac, I made this module using a borrowed one :(
